src/Compute/Compute/help/Add-AzVMSecret.md

```diff
@@ -27,8 +27,28 @@ For more information about the cmdlets, see [Azure Key Vault Cmdlets](/powershel
 
 ## EXAMPLES
 
-### Example 1: Add a secret to a virtual machine
+### Example 1: Add a secret to a virtual machine using the Azure Key Vault virtul machine extension
```

Comment:
I believe the extension is called Azure Key Vault extension, also there is a type in virtual word.

Reply:
Will push and update.